modules/installation-and-upgrade/pages/proxy-uyuni.adoc

```diff
@@ -64,17 +56,10 @@ spacewalk-proxy stop
 At least not for the migration from 15.2 to 153.
 . Click menu:Software[Packages > Upgrade], and select all the packages to be updated on the proxy, and then apply the selection.
 ////
-. Click menu:Details[Remote Command], add `zypper dup --allow-vendor-change` to the script field, and click btn:[Schedule]
+. Click menu:Details[Remote Command], add `zypper --non-interactive dup --allow-vendor-change --replacefiles` to the script field, and click btn:[Schedule]
 . Wait until the remote command is executed.
-. Re-register with the reactivation key using the [command]``rhnreg_ks`` command-line utility.
-  The system is re-registered with the same ID, history, and groups.
-+
-// and channels (unless the system's base channel changes).
-. On the {productname} Proxy, start the proxy service:
+. If required, Re-register with the reactivation key. The system is re-registered with the same ID, history, and groups.
```

Comment:
In my tests after `zypper` command, the proxy was already registered..I would completely remove the re-activation part, but I'm not sure if it can be useful in some particular scenario

Reply:
I am not 100% sure of why this was added. Please use git blame and check with the author. Seems to be @keichwa.

Reply:
acoording to @keichwa re-reg, should not be required ( sometimes a new sometimes install a new proxy instead of upgrading the existing machine: in that case re-reg is required but it's a different use case).  @keichwa is investigating on this and then we'll update the documentation

Reply:
General proxy issues do not belong to this PR; see https://github.com/uyuni-project/uyuni-docs/pull/1560#issuecomment-1141977329 .  Proxy is the topic here: https://github.com/SUSE/spacewalk/issues/17932

Reply:
I'm going to remove all the changes proxy registration (I will keep the `zypper` changes)